ui/scripts/inlineHtml/newUser.js

```diff
@@ -110,7 +110,7 @@ on('clicked:confirm_create_new_user', () => {
     }
     const isRacePicked = selectedRace !== OTHER_RACE;
     setAttrs({
-      ...(isRacePicked ? createBaseSkillsAttributes() : {}),
+      ...createBaseSkillsAttributes(skip_personal = ! isRacePicked),
```

Comment:
This is an unusual syntax. What is going on here.
This may be your bug.
I suspect you want:
```suggestion
      ...createBaseSkillsAttributes(!isRacePicked),
```

Reply:
This was me imagining that javascript supported named arguments in function calls. 

And yet, the code still worked! My argument was interpreted as an assignment expression, so it had the value !isRacePicked. And there was only one parameter for createBaseSkillsAttributes, so it got the intended value.

Reply:
JavaScript does support names arguments in this way:
```
function copyFile({ source, destination }) {
  ...
}

copyFile({ source: 'fileA', destination });
```

Destructuring works well.